packages/datadog_flutter_plugin/test/tracing/tracing_headers_test.dart

```diff
@@ -132,8 +134,10 @@ void main() {
         context.traceId.asString(TraceIdRepresentation.hex32Chars);
     final spanString =
         context.spanId.asString(TraceIdRepresentation.hex16Chars);
-    final expectedHeader = '00-$traceString-$spanString-00';
+    final expectedParentHeader = '00-$traceString-$spanString-00';
+    const expectedStateHeader = 's:0;o:rum';
```

Comment:
just to double check, sampling is required field right?

Reply:
Yes, it's always either 0 or 1.